src/main/java/duke/Action.java

```diff
@@ -0,0 +1,6 @@
+package duke;
+
+/** The Action enum enumerates all possible actions. */
+public enum Action {
```

Comment:
I like how you've used enums to restrict the domain of the actions that can be carried out :)